src/heap/tst/HeapApiFunctionalityTest.cpp

```diff
@@ -136,7 +143,7 @@ VOID minBlockFitAllocResize(PHeap pHeap)
     UINT64 retSize, setSize;
 
     // Set to default
-    MEMSET(handles, INVALID_ALLOCATION_HANDLE_VALUE, NUM_ITERATIONS);
+    zeroHandleArray(handles, NUM_ITERATIONS);
```

Comment:
Why are we replacing `MEMSET()` with `zeroHandleArray()`?

`MEMSET(handles, INVALID_ALLOCATION_HANDLE_VALUE, NUM_ITERATIONS * SIZEOF(ALLOCATION_HANDLE)` should do it, right? DO we need `zeroHandleArray()`?

Reply:
The main issue is that MEMSET operates on byte granularity. INVALID_ALLOCATION_HANDLE_VALUE is evaluated to UINT64. We can't assume that the value is 0 - it can be any arbitrary number. Imagine if invalid handle value was 0x01020304050507. Using MEMSET would set the entire array to 0x07